### ISSUE TITLE
Bug - add lang

### ISSUE THREAD

**Issue by anielka41** [juzaweb/cms#219]
Hi,
it shows an error when adding a new language.
mkdir(): No such file od directory

**Comment by juzaweb** [comment]
Please check your folders read/write permissions and try again
```
plugins/**/src/resources/lang
resources/lang
modules/Backend/resources/lang
```
Let me know, if it still doesn't work

**Comment by anielka41** [comment]
Localhost - CHMOD allow you to write and read.

**Comment by juzaweb** [comment]
https://github.com/juzaweb/translation
Is this the plugin you are using?

**Comment by anielka41** [comment]
Yes :)

**Comment by juzaweb** [comment]
ok, thanks for report, I will check it again

**Comment by juzaweb** [comment]
Fixed in version v3.2.5
